tools/checked-c-convert/ProgramInfo.h

```diff
@@ -120,15 +163,16 @@ class ProgramInfo {
 
   // For each pointer type in the declaration of D, add a variable to the 
   // constraint system for that pointer type. 
-  bool addVariable(clang::Decl *D, clang::DeclStmt *St, clang::ASTContext *C);
+  bool addVariable(clang::DeclaratorDecl *D, clang::DeclStmt *St, clang::ASTContext *C);
 
   bool getDeclStmtForDecl(clang::Decl *D, clang::DeclStmt *&St);
 
   // Checks the structural type equality of two constraint variables. This is 
   // needed if you are casting from U to V. If this returns true, then it's 
```

Comment:
I think that's still what this function does. I just refined the type from Decl to DeclaratorDecl.